hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/SchemaTransaction.java

```diff
@@ -469,8 +470,16 @@ private static Id asyncRun(HugeGraph graph, HugeType schemaType,
             try {
                 task.get();
                 assert task.completed();
+            } catch (ExecutionException e) {
+                Throwable cause = e.getCause();
+                if (cause instanceof RuntimeException) {
+                    throw (RuntimeException) cause;
+                }
+                throw new HugeException("Async task failed with error: %s",
+                                        cause.getMessage(), cause);
```

Comment:
'cause' is exception arg, 'cause.getMessage()' is message arg, so change order of them.

```
         throw new HugeException("Async task failed with error: %s",
                                 cause, cause.getMessage());
```